docs/guides/modules/ROOT/partials/execution-resources/xcode-silicon-vm.adoc

```diff
@@ -8,12 +8,12 @@
 | Release Notes
 
 | `26.1.0`
-| Xcode 26.1 Beta 3 (17B5045g)
+| Xcode 26.1 RC (17B54)
 | 15.6
 a| `m4pro.medium` +
    `m4pro.large`
-| link:https://circle-macos-docs.s3.amazonaws.com/image-manifest/v15774/manifest.txt[Installed software]
-| link:https://circleci.com/changelog/xcode-26-1-beta-3-available/[Release Notes]
+| link:https://circle-macos-docs.s3.amazonaws.com/image-manifest/v15780/manifest.txt[Installed software]
+| link:https://circleci.com/changelog/xcode-26-1-rc-available/[Release Notes]
```

Comment:
The changelog entry hasn't been published yet. This link (https://circleci.com/changelog/xcode-26-1-rc-available/) gives a `404`.

Reply:
weird. i published it yesterday but it's not showing up. i just unpublished and published again we'll see if that works